nexusutils/nexus/nexus.py

```diff
@@ -1138,9 +1237,14 @@ def main():
     logger = logging.getLogger(__name__)
     logger.addHandler(stdout_handler)
     logger.setLevel(logging.DEBUG)
-    nexus_helper = HandleNexus(logger, sys.argv[1:])
+    if documentation and concept:
+        raise ValueError("Only documentation (-d) or is_a relation with a concept (-c)"
```

Comment:
Consider adding `Only one of` or something of the sorts. Just the `or` does not seem sufficient. I guess the idea is to tell users that they can only have either -d or -c but not both.